.github/workflows/create-demo-clusters.yml

```
@@ -62,6 +62,31 @@ jobs:
           PROPERTIES_URL: /repos/${{ github.repository }}/contents/.github/properties?ref=${{ github.ref_name }}
         run: gh api -H "$ACCEPT_RAW" "$PROPERTIES_URL" >> "$GITHUB_OUTPUT"
 
+  parse-refs:
+    name: Parse git refs
+    runs-on: ubuntu-latest
+    outputs:
+      version-ref: ${{ steps.parse.outputs.version-ref }}
+      burner-ref: ${{ steps.parse.outputs.burner-ref }}
+    steps:
+      - name: Parse refs (extract SHA from git-describe format)
+        id: parse
+        run: |
+          parse_ref() {
```

Maybe we should stop the workflow if there are any errors?
```suggestion
          set -euo pipefail
          parse_ref() {
```